windows/lt-base-windows-dotnet.json

```diff
@@ -51,9 +51,9 @@
       "type": "powershell",
       "inline": [
         "choco install 7zip --version 19.0",
-        "choco install nuget.commandline --version 5.8.0",
-        "choco install sonarscanner-msbuild-net46 --version=5.1.0.28487",
-        "choco install sonarscanner-msbuild-netcoreapp2.0 --version=5.1.0.28487",
+        "choco install nuget.commandline --version 5.9.1",
+        "choco install sonarscanner-msbuild-net46 --version=5.2.0.29862",
+        "choco install sonarscanner-msbuild-netcoreapp2.0 --version=5.2.0.29862",
         "choco install visualstudio2019buildtools --version 16.8.2.0 --package-parameters \"--allWorkloads --includeRecommended --includeOptional\"",
```

Comment:
VS 2019 16.8 is affected by [CVE-2021-28322](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2021-28322) - should update to 16.9.4.0

Reply:
(actually, I'm not sure if the build tools are affected by Diagnostics hub...)